### ISSUE TITLE
Uninstalling n on OSX

### ISSUE THREAD

**Issue by mdanetzky** [tj/n#441]
I'm using MacOS Sierra and n 2.1.4 installed using homebrew. To have a clean uninstall I need to remove all installed versions of node. I can't do that, because n rm tells me that
```
Error: cannot remove currently active version
```
It would be great do have an option to deactivate n (just like in nvm).

**Comment by troy0820** [comment]
Could you use the template so I can understand where everything is?  I honestly wouldn't use brew to install n, as it's not in the documentation.  Brew installs in /usr/local/Cellar and symlinks it.

**Comment by mdanetzky** [comment]
I used:
```
brew install n
```
Unfortunately every npm execution needs elevated privileges - which is the reason why I want to remove it. Nonetheless I can imagine, that a functionality to deactivate n might be useful in other scenarios too. For example while switching between various installations.

PS. I just used sudo rm to remove everything that resembled node, npm or node_modules from my system.
